modules/init.txt

```diff
@@ -38,3 +38,5 @@
 
 custom/commands/
 custom/lua/test_npcs_in_gm_home.lua
+ASB/
+era/lua/era_quest_timers.lua
```

Comment:
lets take the quest timers out of the init for now. We can leave this custom so if people want it in they can comment the entire era folder themselves